tests/dbus/test_dbus_client.cpp

```diff
@@ -221,6 +237,8 @@ int main()
         stepDone = true;
     });
 
+    CheckFeatureFlagUpdate(api.get());
+
```

Comment:
Why the test case is invoked here?

Reply:
I added here for the Feature Flag's set/get test. Meanwhile I applied default flags here and want to make sure the system works under default flags. Do you have some suggestions?